### ISSUE TITLE
[Bug]: Modal Produces Double Scrollbars When Content Overflows Container

### ISSUE THREAD

**Issue by cabillin** [microsoft/fluentui#23388]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.25126
    CPU: (24) x64 AMD Ryzen Threadripper PRO 3945WX 12-Cores
    Memory: 33.93 GB / 63.86 GB
  Browsers:
    Edge: Spartan (44.25126.1000.0), Chromium (102.0.1245.30)
    Internet Explorer: 11.0.25126.1000
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/KnetTravlr/pen/yLvEMqb

## Actual Behavior
Double scrollbars when modal content overflows container (e.g. attached codepen--https://codepen.io/KnetTravlr/full/yLvEMqb--at 400% zoom). 
![image](https://user-images.githubusercontent.com/71525014/171777509-9b7c1128-6cd8-40e3-9f99-62134fe3053a.png)

This is a problem for both x and y directions.

I specified that this is an accessibility bug because it impacts reflow.

## Expected Behavior
Single scrollbar to deal with overflow

### Requested priority

Normal

### Products/sites affected

internal Microsoft website

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by levani132** [comment]
This problem comes from the scrollable content having max height 100vh I believe, it should have calc(100vh - 32px).

https://github.com/microsoft/fluentui/blob/f11b365e857e7f01879f0a597c2496821a844ada/packages/react/src/components/Modal/Modal.styles.ts#L101

It causes some problems for me too, though I have this workaround of passing `scrollableContentClassName="max-h-[calc(100vh-32px)]"` as input. Although we're using tailwind, I think you can guess what I'm doing, and this can be done using usual class and pure css/scss too.

**Comment by cabillin** [comment]
Please reopen, or mark as dupe of https://github.com/microsoft/fluentui/issues/27369 or https://github.com/microsoft/fluentui/issues/12420 and keep the chosen one open. This issue continues to be a problem and impacts a11y. @mishabutuzov provided a work around, but this issue should remain open until the component's default styles are correct.

**Comment by khmakoto** [comment]
@cabillin, I'm marking this as a duplicate of #27369, thanks for already tracking down an open issue.
